.changeset/five-mirrors-join.md

```diff
@@ -0,0 +1,21 @@
+---
+'xstate': major
+'@xstate/fsm': major
+'@xstate/react': major
+'@xstate/vue': major
+---
+
+The output data on final states is now specified as `.output` instead of `.data`:
+
+```diff
+const machine = createMachine({
+  // ...
+  states: {
+    // ...
+    success: {
+-     data: { message: 'Success!' }
++     output: { message: 'Success!' }
```

Comment:
this later becomes part of the `done.state.*` and `done.invoe.*` events, should we rename the properties in those events as well?

Reply:
I'm not sure. I think there should be a difference between "this is the output of this machine" and "this is the data in this event". Not sure yet.